client/src/pages/TestComponents.tsx

```diff
@@ -0,0 +1,166 @@
+import {
+  HoverCard,
+  HoverCardContent,
+  HoverCardTrigger,
+} from '@/components/ui/hover-card';
+import clsx from 'clsx';
+import * as littlethingsApi from '@/api-client/littlethingsApi';
+import { Dispatch, SetStateAction } from 'react';
+import { Frequency, options } from '@/api-client/homepageutility';
+import { Post } from '@/api-client/homepageutility';
+
+interface CardComponentProps {
+  setPosts: Dispatch<SetStateAction<Post[]>>;
+  littlething: string;
+  description: string;
+  frequency: Frequency;
+  occurence: number;
+  createdAt: string;
+  user_id?: string;
+  updatedAt?: string;
+  postId: string;
+  posts: Post[];
+  deletable?: boolean;
+}
+
+const TestComponents = ({
+  setPosts,
+  littlething,
+  description,
+  frequency,
+  occurence,
+  createdAt,
+  postId,
+  deletable,
+}: CardComponentProps) => {
+  const deletePostHandler = async (postId: string) => {
+    const success = await littlethingsApi.deletePost(postId);
+    if (success) {
```

Comment:
needs error handling/user feedback on error

Reply:
will do so later, user will go through confirm delete pop up and error will be displayed there